### ISSUE TITLE
Question about reproducing ScanQA results

### ISSUE THREAD

**Issue by ZechuanLi** [LZ-CH/DSPNet#8]
Hi, thanks for your great work. DSPNet is a very impressive method.

I tried to reproduce the results using the default configs for both SQA3D and ScanQA. The SQA3D results are close to those reported in the paper, but the ScanQA results show a relatively large gap.

I noticed that the ScanQA config seems to use the **with detection** setting, but my best reproduced results are still lower than the reported **w/o detection** results:

- EM@1: 0.2233
- EM@10: 0.5301

Could you please let me know if any config changes, specific checkpoints, or additional preprocessing steps are needed for ScanQA?
Thanks!

**Comment by LZ-CH** [comment]
When training, did you use the same number of GPUs and batch size as specified in our paper? Furthermore, do the results you obtained correspond to the test set or the validation set? Additionally, the results presented in the paper were obtained by loading pre-trained VoteNet weights.
